ui/src/routes/_layout/organizations/$orgId/products/$productId/repositories/$repoId/_layout/runs/$runIndex/vulnerabilities/index.tsx

```diff
@@ -28,16 +49,205 @@ import {
+      <div className='italic text-muted-foreground'>
+        <div className='font-semibold'>Description:</div>
+        {row.original.vulnerability.description}
+      </div>
+      <Accordion type='single' collapsible className='ml-4'>
+        <AccordionItem value='references' className='rounded-lg border px-2'>
+          <AccordionTrigger>
+            <Label className='font-semibold text-blue-400 hover:underline'>
+              References
+            </Label>
+          </AccordionTrigger>
+          {references.map((reference, index) => (
+            <AccordionContent key={index}>
+              <div className='flex gap-2'>
+                <div>{reference.severity}</div>
+                <div>{reference.scoringSystem}</div>
+                <Link
+                  className='font-semibold text-blue-400 hover:underline'
+                  to={reference.url}
+                  target='_blank'
+                >
+                  {reference.url}
+                </Link>
+              </div>
+            </AccordionContent>
+          ))}
+        </AccordionItem>
+      </Accordion>
+    </div>
+  );
+};
 
 const VulnerabilitiesComponent = () => {
+  const params = Route.useParams();
+  const search = Route.useSearch();
+  const pageIndex = search.page ? search.page - 1 : 0;
+  const pageSize = search.pageSize ? search.pageSize : defaultPageSize;
+
+  const { data: ortRun } = useRepositoriesServiceGetOrtRunByIndexSuspense({
+    repositoryId: Number.parseInt(params.repoId),
+    ortRunIndex: Number.parseInt(params.runIndex),
+  });
+
+  const {
+    data: vulnerabilities,
+    isPending,
+    isError,
+    error,
+  } = useVulnerabilitiesServiceGetVulnerabilitiesByRunId({
+    runId: ortRun.id,
+    limit: pageSize,
+    offset: pageIndex * pageSize,
+  });
+
+  const table = useReactTable({
+    data: vulnerabilities?.data || [],
+    columns,
+    pageCount: Math.ceil(
+      (vulnerabilities?.pagination.totalCount ?? 0) / pageSize
+    ),
+    state: {
+      pagination: {
+        pageIndex,
+        pageSize,
+      },
+    },
+    getCoreRowModel: getCoreRowModel(),
+    getRowCanExpand: () => true,
+    getExpandedRowModel: getExpandedRowModel(),
+    manualPagination: true,
+  });
+
+  if (isPending) {
+    return <LoadingIndicator />;
+  }
+
+  if (isError) {
+    return error;
+  }
+
   return (
     <Card className='h-fit'>
       <CardHeader>
-        <CardTitle>Vulnerabilities</CardTitle>
-        <CardDescription>Vulnerabilities of the project.</CardDescription>
+        <CardTitle>Vulnerabilities (ORT run global ID: {ortRun.id})</CardTitle>
+        <CardDescription>
+          These are the vulnerabilities found from the project. By clicking on
+          "References" you can see more information about the vulnerability.
+          Please note that in the initial implementation the overall severity
+          rating is calculated based on the highest severity rating found in the
+          references.
```

Comment:
"Found **in** the project" is probably more correct.

I'd state that the highest severity is shown without talking about it being an initial implementation.

Reply:
No, it's the highest rating of the references, really.